sessionctx/variable/sysvar.go

```diff
@@ -164,7 +164,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeGlobal, "innodb_max_undo_log_size", ""},
 	{ScopeGlobal | ScopeSession, "range_alloc_block_size", "4096"},
 	{ScopeGlobal, ConnectTimeout, "10"},
-	{ScopeGlobal | ScopeSession, "collation_server", charset.CollationUTF8},
+	{ScopeGlobal | ScopeSession, "collation_server", mysql.DefaultCollationName},
```

Comment:
I found `charset.CharsetUTF8, charset.CollationUTF8` was refered in another pkg, e.g. `expression/ builtin_*`, does them also need change?

Reply:
I changed it in more places now :)